### DISCUSSION TITLE
Link for Policy Authoring Guide should be on What Is cnspec? Page

### DISCUSSION THREAD

**Discussion by kmf** [mondoohq/cnquery#1189] [Ideas]
I think a link to the [Policy Authoring Guide](https://mondoo.com/docs/cnspec/cnspec-policies/write/) should be on the [What Is cnspec?](https://mondoo.com/docs/cnspec/cnspec-about/) page

**Comment by chris-rock** [comment]
@misterpantz I think that is a great idea

**Comment by misterpantz** [comment]
Ooh yes. I need to add links to the Policy Authoring Guide from a few places! I'll be sure to include the What Is page. Thanks, @kmf and @chris-rock!

**Comment by misterpantz** [comment]
This is live now. Thank you again!
